paasc/src/main.rs

```diff
@@ -0,0 +1,56 @@
+use std::{fs::File, io::BufReader};
+
+use rustls::{ciphersuite, internal::pemfile, ClientConfig, RootCertStore, SupportedCipherSuite};
+use tonic::transport::Channel;
+
+use paas_types::process_service_client as client;
+use paas_types::StatusRequest;
+
+static CIPHERSUITES: &[&SupportedCipherSuite; 5] = &[
+    &ciphersuite::TLS13_AES_256_GCM_SHA384,
+    &ciphersuite::TLS13_CHACHA20_POLY1305_SHA256,
+    &ciphersuite::TLS13_AES_128_GCM_SHA256,
+    &ciphersuite::TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
+    &ciphersuite::TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+];
+
+fn buf_read(path: &str) -> Result<BufReader<File>, Box<dyn std::error::Error>> {
```

Comment:
Since you use the error type a few times, import it or make an alias or alternatively use something like anyhow to get a less complex type to read.

Reply:
Migrated `paasc` to `anyhow::Error` in an in-progress branch (future PR). Hope that is alright.

Reply:
great, thanks for the explanation